translation-web/src/main/java/eu/europeana/api/translation/web/service/TranslationWebService.java

```diff
@@ -98,7 +102,7 @@ private TranslationService getTranslationService(final String serviceId,
     String param = fallback ? TranslationAppConstants.FALLBACK : TranslationAppConstants.SERVICE;
     if (result == null) {
       throw new ParamValidationException(null, I18nConstants.INVALID_SERVICE_PARAM,
-          new String[] {param, serviceId});
+          new String[] {param, serviceId + " (valid values: " + TranslationAppConstants.PANGEANIC + ", " + TranslationAppConstants.GOOGLE + ")"});
```

Comment:
the list of available services (please replace also the message) should include the keys of the service maps available in the TranslationServiceProvider